### PR TITLE
Add context when failing to map a db measure to a ws measure

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/measure/ws/MeasureDtoToWsMeasure.java [hunk 1]
```diff
@@ -33,24 +33,28 @@ private MeasureDtoToWsMeasure() {
...
+            .setIndex(i)
+            .setValue(formatNumericalValue(measureDto.getVariation(i), metricDto)));
+        }
       }
-    }
 
-    return measure.build();
+      return measure.build();
+    } catch (Exception e) {
+      throw new IllegalStateException(String.format("Error while mapping a measure of metric key '%s' and characteristics %s", metricDto.getKey(), measureDto.toString()), e);
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/measure/ws/MeasureDtoToWsMeasure.java]: characteristics => measure ?

Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/measure/ws/MeasureDtoToWsMeasure.java]: Rephrase the sentence "Error while mapping measure '%s' of metric ''%s" ?

Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/measure/ws/MeasureDtoToWsMeasure.java]: I had difficulties. I'm talking about measure parameters

**Comment by julienlancelot** [comment]
Apply feedback then LGTM

**Pull request merged**